swagger_server/controllers/operand_controller.py

```diff
@@ -35,10 +51,14 @@ def get_operand(operand_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    try:
+        operand = Operand.get_by_id(operand_id)
+    except Exception as exc:
+        raise ValidationError(400, 'Expression not found!')
+    return flask.jsonify({'id': operand_id})
```

Comment:
We should serialize the whole object like https://github.com/math-as-a-service/configurations/pull/17/files#diff-9ebe98c8418892f4fa60558fe9083195R27